context.go

```diff
@@ -47,15 +52,20 @@ type LogContext interface {
 //////// SETUP:
 
 // Creates a new Context with an empty dispatch table.
-func NewContext() *Context {
-	return NewContextCustomID(fmt.Sprintf("%x", rand.Int31()))
+func NewContext(AppProtocolId string) *Context {
```

Comment:
Shouldn't this be lowercase? It's a function parameter. (Ditto for line 63.)

Reply:
Thanks for catching.  Fixed and pushed.